server.h

```diff
@@ -204,6 +209,20 @@ class CTFOServer {
             } else {
               card_entry.ctfo_percentage = 0.5;
             }
+          };
+
+          for (size_t i = 0; i < candidates.size() / 2 && (max_count ? (i < max_count) : true); ++i) {
+            const CID hot_cid = candidates[i];
```

Comment:
If your response would contain a `map<string, Feed>`, it would be `const char* feed_keys[2] = { "hot", "recent" }; for (...) { auto& feed = feeds[feed_keys[i % 2]]; ... }` :-)